dev-itpro/developer/methods-auto/jsonobject/jsonobject-add-string-string-method.md

```diff
@@ -48,6 +48,78 @@ An instance of the [JsonObject](jsonobject-data-type.md) data type.
 ## Remarks
 The operation will fail if the object already contains a property with the given key.
 
+Note that text constants, such as labes or raw text values assigned in the source code, will be transformed by the compiler. All characters that require escaping in C# strings will be prefixed with the backslash symbol ('\\'). These characters include single and double quotes, backslashes, control characters (for example, line feed or tab). For more information, refer to [String escape sequences](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#string-escape-sequences).
```

Comment:
@NKarolak Thank you for the recommendations, I updated the link. Let's see what the bot has to say now. :-)

Reply:
@adrogin + @NKarolak - thanks both - the /en-us/ needs to go, yes, and then we use the relative path for closed environments - in this case, I've changed it to /dotnet/.... which will be valid, because we're already on the learn site.